src/Box.tsx

```diff
@@ -39,29 +61,56 @@ export function Box({ mapImage, normalImage }: Props) {
     mirrorWrapY: {
       value: false,
     },
-  })
+  });
+
+  const placeholderTexture = new Texture();
+  placeholderTexture.image = new Image();
+
+  const createTexture = (
+    image: HTMLImageElement | null,
+    wrapX: boolean,
+    wrapY: boolean
+  ): Texture | null => {
+    if (image) {
```

Comment:
@parthvaghani if there is no images the map should be null, using an empty texture might be the reason for the black cube